freqtrade/plugins/pairlistmanager.py

```diff
@@ -108,9 +108,11 @@ def verify_blacklist(self, pairlist: List[str], logmethod) -> List[str]:
         except ValueError as err:
             logger.error(f"Pair blacklist contains an invalid Wildcard: {err}")
             return []
-        for pair in deepcopy(pairlist):
+        for pair in pairlist.copy():
             if pair in blacklist:
-                logmethod(f"Pair {pair} in your blacklist. Removing it from whitelist...")
+                if pair not in self._logged_blacklist_pairs:
+                    logmethod(f"Pair {pair} in your blacklist. Removing it from whitelist...")
+                    self._logged_blacklist_pairs.add(pair)
```

Comment:
I am not sure i like this kind of tracking / log surpression, considering freqtrade is potentially started, and then potentially "left running" for weeks / months at a time.

This can lead to a situation where you're running freqtrade for 2 weeks - and start wondering why a pair is not in your pairlist - since the logs "don't contain anything" (you'll not look through 2 weeks of logs - but only through the last half a day or so).


you should instead look at the `LoggingMixin` instead - which is used in the pairlist providers themselvs. 
it will reduce the amount of log messages, but will still log it from time to time - which i think is a better approach, and is aligned to how this works for all other pairlists, without reinventing the wheel.
this can potentially even collide with how VolumePairlist uses logging (see below).

`VolumePairlist` already implements blacklist verification logging this way.

https://github.com/freqtrade/freqtrade/blob/dda302eea28d7e4d3bad3615627554d9a76975ad/freqtrade/plugins/pairlist/VolumePairList.py#L207

Reply:
Updated as per your request / recommendation. The tests passed as well (no changes required).

Reply:
Strange that tests passed for MacOS X and Windows, but not on Ubuntu.
I've run pytest on Ubuntu 20.04 (latest update) on Windows/WSL2, and it worked perfectly...

Reply:
pytest in CI runs with randomized test-sequence - so it's not a systematic failure on ubuntu - but with this particular seed.